www/front_src/src/Resources/index.test.tsx

```diff
@@ -5,6 +5,7 @@ import formatISO from 'date-fns/formatISO';
 import { render, waitFor, within, fireEvent } from '@testing-library/react';
 import UserEvent from '@testing-library/user-event';
 import last from 'lodash/last';
+import { Simulate } from 'react-dom/test-utils';
```

Comment:
user-event currently does not manage special keys.
Waiting this PR : https://github.com/testing-library/user-event/pull/235